### DISCUSSION TITLE
Erro ao Debug

### DISCUSSION THREAD

**Discussion by ghost** [balta-io/2801#15] [Perguntas e Respostas] [answered]
![erro](https://user-images.githubusercontent.com/90652272/136481388-8bdbcb13-395f-4d1c-92fd-b2ac34ea616c.jpg)

Estou nas aulas iniciais e estou enfrentando esse erro ao tentar debug.

**Comment by BrewertonSantos** [comment, marked as answer]
Boa tarde @WallaceHerlon!

Notei que na imagme enviada o depurar selecionado é o `.NET Core Launcher`. Na parte superior esquerda da sua janela do Visual Studio Code, Clique onde está escrito o nome do depurador e selecione o `.NET Core Attach`.

![image](https://user-images.githubusercontent.com/55927647/136627176-0b12898c-1d8d-4b14-9392-046b22195cc5.png)

Verifique se a solução proposta acima funciona.

Tente também reinstalar o .NET, de preferencia o 6: https://dotnet.microsoft.com/download/dotnet/6.0

**Reply by BrewertonSantos** [reply]
Após reinstalar o pacote .NET o problema do @WallaceHerlon foi corrigido.

**Reply by andrebaltieri** [reply]
@BrewertonSantos muito obrigado <3

**Comment by renatoidev** [comment]
Estou tendo problemas para debugar também.

Acho que meu código não está entrando no if da regex, tentei debugar pra ver se entrava lá mas deu esse erro
![image](https://user-images.githubusercontent.com/70719134/150128199-b9a69fa6-09ab-42d5-9299-3185d0c0e068.png)

meu código tá nesse link https://github.com/renatoidev/EditorHTML

tentei mudar a opção para o attached igual está falando para o outro colega, ele pede para selecionar um diretório, eu coloquei o dotnet mas não rodou, tentei reinstalar o dotnet também para o 6.0, igual indicado, porém também não deu certo.

quem puder me ajudar agradeço muito.

**Reply by BrewertonSantos** [reply]
Olá @renatoidev! Pode exibir por gentileza o seu arquivo de consiguração do debugger?

**Reply by andrebaltieri** [reply]
@renatoidev uma sugestão, executa os comandos abaixo na raiz do seu App.
```
dotnet clean
dotnet build
```

Isto vai excluir todas as DLLs e gerar novamente. A partir daí, tenta debugar novamente.
As vezes o Windows deixa algumas coisas no cache e ocorre estes tipos de erro.

**Reply by renatoidev** [reply]
Em qual pasta fica a configuração do debugger? Nas pastas obj e debugger desse link não tem elas?https://github.com/renatoidev/EditorHTML

tentei fazer o clean e build mas também não rodou o debug.

Eu fiz uns testes usando o visual studio copiando as classes igualzinho e rodou certinho, inclusive o debug funcionou, fiz o teste de rodar a aplicação direto no windows terminal e funcionou beleza também.

Acho que o o problema ta no vs code.

**Reply by andrebaltieri** [reply]
@renatoidev tem uma pasta chamada `.vscode` e outra chamada `Properties`, normalmente ficam nela.
Pode ser algo do Visual Studio Code também, tem algum erro na tela, no canto inferior direito? Normalmente quando tem alguma inconsiostência, aparece ali!

**Reply by renatoidev** [reply]
![image](https://user-images.githubusercontent.com/70719134/150801452-43d710c1-d7ad-4e9c-8cc8-848f2c6468c5.png)
não aparece erro lá embaixo.

mas como consegui pelo visual studio não tem problema. Vou dar sequencia aos cursos e caso dê algum outro problema eu volto a perguntar aqui. 

brigadão pela ajuda

**Reply by andrebaltieri** [reply]
Perfeito @renatoidev <3
